### PR TITLE
Remove the old reference link

### LINKED ISSUE #19152: docker userguide need update for related link about volume part
In this guide
https://github.com/docker/docker/blob/master/docs/userguide/dockervolumes.md

It still refers 

    Note: Docker will not warn you when removing a container without providing the -v option to delete its volumes. If you remove containers without using the -v option, you may end up with "dangling" volumes; volumes that are no longer referenced by a container. Dangling volumes are difficult to get rid of and can take up a large amount of disk space. We're working on improving volume management and you can check progress on this in pull request #14214

But the pull request was closed,  so either a new issue be linked with or remove that link

### PR DESCRIPTION
**Pull request by HackToday**
Fixes: #19152

As issue #14214 closed, so this old link is not needed.
We can remove it now.

Signed-off-by: Kai Qiang Wu(Kennan) wkqwu@cn.ibm.com

### CODE REVIEW

**Comment by HackToday** [comment]
ping @thaJeztah

File: docs/userguide/dockervolumes.md [hunk 1]
```diff
@@ -236,9 +236,8 @@ allows you to upgrade, or effectively migrate data volumes between containers.
 > providing the `-v` option to delete its volumes. If you remove containers
 > without using the `-v` option, you may end up with "dangling" volumes;
 > volumes that are no longer referenced by a container.
-> Dangling volumes are difficult to get rid of and can take up a large amount
-> of disk space. We're working on improving volume management and you can check
-> progress on this in [pull request #14214](https://github.com/docker/docker/pull/14214)
+> You can use `docker volume ls -f dangling=true` to check your dangling volumes.
+> And use `docker volume rm <volume name>` to remove it.
```
Comment (thaJeztah) [inline docs/userguide/dockervolumes.md]: I think that should be one sentence. Perhaps;

```Markdown
> You can use `docker volume ls -f dangling=true` to find dangling volumes,
> and use `docker volume rm <volume name>` to remove a volume that's
> no longer needed.
```

wdyt?

**Comment by thaJeztah** [comment]
Nice! Added one suggestion :)

**Comment by HackToday** [comment]
@thaJeztah  PTAL. Thanks

**Comment by thaJeztah** [comment]
cool, thanks!

LGTM

ping @moxiegirl for review :)

**Comment by MHBauer** [comment]
Reads well, good update.

**Comment by moxiegirl** [comment]
@HackToday Thanks for the contribution. LGTM

**Comment by Aliance** [comment]
Why I still see out-of-dated information at you [manual](https://docs.docker.com/engine/userguide/dockervolumes/) page?

**Comment by thaJeztah** [comment]
@Aliance the GitHub repository contains changes that are not published yet; next update of the online documentation will be together with the 1.10 release

**Pull request merged**